src/orangeClockFunctions/displayBlockMoscowFees.py

```diff
@@ -70,6 +70,14 @@ def getMoscowTime():
     return moscowTime
 
 
+def getFiatPrice(currency):

```

Comment:
A more typical name would be something like "getPriceDisplay" (I think this is my preferred option) or "getPriceStr" to make it clear that it's related to "getPrice" but it returns a formatted string.